tcms/testruns/models.py

```diff
@@ -127,12 +127,12 @@ def belong_to(self, user):
 
         return False
 
-    def check_all_case_runs(self, case_run_id=None):
-        tcrs = self.case_run.all()
-        tcrs = tcrs.select_related('case_run_status')
+    def check_all_case_runs(self):
+        test_case_runs = self.case_run.all()
+        test_case_runs = test_case_runs.select_related('case_run_status')
```

Comment:
FYI you can squash these if they fit on a single lline. Also the order is usually select_related() and then all()